commerce_coordinator/apps/titan/tests/test_clients.py

```diff
@@ -1,170 +1,140 @@
 """Test Titan clients."""
 
-import json
 from urllib.parse import urljoin
```

Comment:
is there a reason to use this over your URL Join, thats the unclear part to us. Doesnt block merge.